contrib/urls-maintenance.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/sh
+# operates various maintenance tasks on urls and cache.db for newsboat:
+# - convert http to https in urls file when https feed is available
+# - convert feeds urls to their new forwarded urls for 3xx http codes
+# - feeds urls not returning a valid http code 200 are tagged "_dead"
+# cache.db is also updated to refer to the new url feeds so all read/unread articles and flags are preserved
+
+# TODO
+# implement additional checks for "valid" feeds returning 200 http code: 
+# https://www.linuxjournal.com/content/parsing-rss-news-feed-bash-script
+#	is it returning valid rss?
+#	when was the feed last updated?
+#	sort valid feeds by last updated
+#	tag feed "abandoned" when feed not updated for more than 1 year
+
+
+#newsboat urls file and cache locations
+u="$HOME/.config/newsboat/urls"
+db="$HOME/.config/newsboat/cache.db"
```

Comment:
Is this a typo? The cache [should go](https://newsboat.org/releases/2.16.1/docs/newsboat.html#_xdg_base_directory_support) to `~/.local/share/newsboat`.

Reply:
right i leave the cache there to speed up cleaning after tests

Reply:
Ah, okay. I'll keep this comment unresolved for now, so we don't forget to address it before merging this.